docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -225,12 +235,17 @@ public int getSSHLaunchTimeoutMinutes() {
             }
         }
     }
+
     /**
      * Initializes data structure that we don't persist.
      */
     protected Object readResolve() {
         super.readResolve();
 
+//        if (launcher != null) {
```

Comment:
I know you do, but we all can benefit with a reminder comment in a pull request... I've been known to put such reminders in myself **on my own pull requests** so that I don't forget. This is a big change... a good change, but still big. There is lots that anyone could forget, reminders are a good thing... but if you're going to get all tetchy maybe I will stop reviewing this change altogether.